Source/JSQSystemSoundPlayer.m

```diff
@@ -130,31 +128,41 @@ - (void)playSoundWithName:(NSString *)filename
     NSParameterAssert(filename != nil);
     NSParameterAssert(extension != nil);
 
-    if (!self.on) {
-        return;
-    }
```

Comment:
@jessesquires 
now it will check ```self.on```  -twice- if you play a normal file, want to extract a new method, but struggling on method name... have a good idea?
cheers